docs/serializers.rst

```diff
@@ -1,4 +1,73 @@
 .. _serializers:
 
 Serializers
-===========
\ No newline at end of file
+===========
+
+Returned data from resource is firstly serialized to python format via serializers. Serialization is performed recursively for complex data types (list, dict, tuple, django Model, django Queryset, etc.). Next step of serialization is converting python format response to the required format (JSON, XML, XLSX, etc.). For this purpose is used :ref:`converters`.
+
+There are several serializers every is used to serialize different type of data to get python format:
```

Comment:
There are several serializers -- each one is used to serialize a different data type to get the Python format.